lib/newgistics/version.rb

```diff
@@ -1,3 +1,3 @@
 module Newgistics
-  VERSION = "2.2.0"
+  VERSION = "2.0.2"
```

Comment:
Can we please undo this? We'd like to bump the version in a separate commit :)

Reply:
Yep - was a typo in the revert. Sorry about that!